src/components/result/FilterSelect/index.tsx

```diff
@@ -1,58 +1,56 @@
-import { useMedia } from '@offer-ui/react'
-import { useEffect, useState } from 'react'
 import type { ReactElement } from 'react'
 import { Styled } from './styled'
 import type { FilterSelectProps } from './types'
 import { PriceDialog } from '../PriceDialog'
-
+import { useGetSortOptions } from '@apis/post/queries'
```

Comment:
```suggestion
import { useGetSortOptions } from '@apis'
```
베럴 파일에 직접 접근해도 될 것 같습니다!

Reply:
[완료!](https://github.com/price-offer/offer-fe/pull/174/commits/a6f681de442a40533851b770a8a71280628c236f#diff-b6812d91ff1eb7128a9a65edbe1a879bad0a8c36ead6871520eec2416d7b960c)